src/cart/SConscript

```diff
@@ -22,6 +22,25 @@ SRC = ['crt_bulk.c', 'crt_context.c', 'crt_corpc.c',
        'crt_swim.c', 'crt_tree.c', 'crt_tree_flat.c', 'crt_tree_kary.c',
        'crt_tree_knomial.c', 'crt_hlc.c', 'crt_hlct.c']
 
+LIB_SRC = ['crt_utils.c']
+
+##############################
+# Create cart utility library
+##############################
+def build_utility_library():
+    """Build cart utility library"""
+
+    Import('env', 'prereqs')
+    denv = env.Clone()
+    denv.AppendUnique(CPPPATH=[Dir('.').srcnode()])
```

Comment:
These two lines are not necessary if you move the call to build_utility library down past where they are set in default environment.

Reply:
Moved down.